ee-feature-pack/common/src/main/resources/content/appclient/configuration/appclient.xml

```diff
@@ -118,6 +118,11 @@
                               managed-thread-factory="java:jboss/ee/concurrency/factory/default"/>
         </subsystem>
         <subsystem xmlns="urn:jboss:domain:ejb3:5.0"/>
+        <subsystem xmlns="urn:wildfly:elytron:17.0" initial-providers="elytron" >
```

Comment:
@fjuma I updated this. I didn't add the 'aggregate-provider-combined-providers' resource because I didn't want to risk any disruption before the 28 Beta tag. But if you think it's good to add we can add that before 28 Final.

Reply:
Thanks @bstansberry, sounds good. In general, adding the `combined-providers` and setting `final-providers=combined-providers` shouldn't cause issues since the `openssl` provider wouldn't actually get used during the test runs but no need to do that right now.